tests/acceptance/features/apiSpaces/notification.feature

```diff
@@ -21,108 +21,159 @@ Feature: Notification
+                    "enum": [
+                      "Alice Hansen"
+                    ]
+                  },
+                  "id": {
+                    "type": "string",
+                    "enim": [
+                      "%user_id%"
+                    ]
+                  },
+                  "name": {
+                    "type": "string",
+                    "enum": [
+                      "Alice"
+                    ]
+                  }
                 }
               }
             }
+          },
+          "notification_id": {
+            "type": "string"
+          },
+          "object_id": {
+            "type": "string"
+          },
+          "object_type": {
+            "type": "string",
+            "enum": [
+              "storagespace"
+            ]
+          },
+          "subject": {
+            "type": "string",
+            "enum": [
+              "Space shared"
+            ]
+          },
+          "subjectRich": {
+            "type": "string",
+            "enum": [
+              "Space shared"
+            ]
+          },
+          "user": {
+            "type": "string",
+            "enum": [
+              "Alice"
+            ]
           }
-        },
-        "notification_id": {
-          "type": "string"
+        }
+      }
+      """
 
-        },
-        "object_id": {
-          "type": "string"
 
-        },
-        "object_type": {
-          "type": "string",
-          "enum": ["storagespace"]
-        },
-        "subject": {
-          "type": "string",
-          "enum": ["Space shared"]
-        },
-        "subjectRich": {
-          "type": "string",
-          "enum": ["Space shared"]
-        },
-        "user": {
-          "type": "string",
-          "enum": ["Alice"]
+  Scenario Outline: user gets a notification of space sharing in different languages
+    Given user "Alice" has created a space "notification checking" with the default quota using the GraphApi
+    And user "Brian" has switched the system language to "<language>"
+    And user "Alice" has shared a space "notification checking" with settings:
+      | shareWith | Brian  |
+      | role      | editor |
+    When user "Brian" lists all notifications
+    Then the HTTP status code should be "200"
+    And the JSON response should contain a notification message with the subject "<subject>" and the message-details should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "message"
+        ],
+        "properties": {
+          "message": {
+            "type": "string",
+            "enum": [
+              "<message>"
+            ]
+          }
         }
       }
-    }
-    """
-
+      """
+    Examples:
+      | language | subject           | message                                                               |
+      | de       | Space freigegeben | Alice Hansen hat Sie zum Space notification checking hinzugef\u00fcgt |
```

Comment:
now the message is `Alice Hansen hat Sie zu Space notification checking hinzugefügt` see https://drone.owncloud.com/owncloud/ocis/22299/52/6
does ocis sometimes encode it and sometimes not?

Reply:
Both cases failed in CI: 

`hinzugefügt`
<img width="730" alt="Screenshot 2023-05-11 at 11 43 31" src="https://github.com/owncloud/ocis/assets/84779829/282dce48-9daa-41a7-8032-022fcf12f0a8">

`hinzugef\u00fcgt
`<img width="745" alt="Screenshot 2023-05-11 at 11 42 59" src="https://github.com/owncloud/ocis/assets/84779829/ef9ad848-1831-44d2-bd03-ed1d92532e63">